app-emulation/cri-tools/cri-tools-1.18.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit coreos-go bash-completion-r1
```

Comment:
`bash-completion-r1` seems to be unnecessary, since we don't install any bash completions.

Reply:
> bash-completion-r1 seems to be unnecessary, since we don't install any bash completions.

ok, will do.

Reply:
Done.